pkg/metastore/kv_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2022-2023 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metastore
+
+import (
+	"testing"
+
+	pb "github.com/parca-dev/parca/gen/proto/go/parca/metastore/v1alpha1"
+)
+
+func TestMakeLocationID(t *testing.T) {
+	tests := map[string]struct {
+		loc  *pb.Location
+		want string
+	}{
+		"one line": {
+			loc: &pb.Location{
+				Address:   9424419,
+				MappingId: "2b-t2tYPDARtdf-_FCsqMnUDllVoG8eHx3DGY6B2zsc=",
+				Lines: []*pb.Line{
+					{
+						FunctionId: "dFRfEsG-uRAep6DtB_p4adjSQiRipfgf0LTZ3B05k74=/auEFFJYKPIUbrC2nw-kBi9ePl80B2bwY6mCyCpUeC78=",
+						Line:       206,
+					},
+				},
+			},
+			want: "2b-t2tYPDARtdf-_FCsqMnUDllVoG8eHx3DGY6B2zsc=/kBL4fDvKSLKe5R_x08kcKOji7UXafFAYjquOeVeZsrA=",
```

Comment:
I grabbed these values (added print statements and ran the tests) from `MakeLocationID` before making changes to it.